ink-wrapper-types/src/aleph_client.rs

```diff
@@ -96,27 +103,30 @@ impl crate::SignedConnection<TxInfo, anyhow::Error> for aleph_client::SignedConn
             .map_err(|e| anyhow!("Code upload failed {:?}", e))?
             .code_hash;
 
-        if actual_code_hash.as_ref() != code_hash {
+        if actual_code_hash.as_ref() != call.expected_code_hash {
             return Err(anyhow!(
                 "Code hash mismatch: expected {:?}, got {:?}",
-                code_hash,
+                call.expected_code_hash,
                 actual_code_hash
             ));
```

Comment:
This will not render nicely:
```
Error: Code hash mismatch: expected [149, 194, 140, 176, 48, 41, 75, 38, 46, 68, 217, 31, 215, 190, 36, 141, 98, 5, 247, 41, 33, 138, 109, 69, 155, 114, 113, 213, 145, 225, 158, 198], got 0xea80939fcccf0ad17d51a068ee5aa6bbf793138910ff558ddb720d83e7eb9580
```
Please fix the string representation while you're at it.